index.php

```diff
@@ -36,6 +37,18 @@
 	HTMLHelper::_('stylesheet', 'switch.css', ['version' => 'auto', 'relative' => true]);
 }
 
+// Mobile menu toggle
+$doc->addScriptDeclaration("
+	function menuToggle() {
```

Comment:
There is an inline script block at the bottom of the file already? Could you put it down there please?